### PR TITLE
Adapt for requires.io - Update versions

### LINKED ISSUE #2070: Consider using requires.io to keep Python dependencies up-to-date
[requires.io](https://requires.io/) is similar to Greenkeeper, it checks if there are news versions for Python dependencies.

### PR DESCRIPTION
**Pull request by sbrunner**
Fix #2070

### CODE REVIEW

**Comment by sbrunner** [comment]
Ready to be review :-)

**Comment by sbrunner** [comment]
Result: https://requires.io/github/camptocamp/c2cgeoportal/requirements/?branch=requires-io

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -741,7 +741,7 @@ endif
 
 $(NPM_CMD):
 ifeq ($(OPERATING_SYSTEM), LINUX)
-	npm install npm
+	npm install npm || npm install npm || npm install npm
```
Comment (fredj) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: not sure about this ...

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Travis failed and refailed :-(

**Pull request merged**